client/package.json

```diff
@@ -3,12 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/react-fontawesome": "^0.1.14",
     "@testing-library/jest-dom": "^5.11.8",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.6.0",
     "node-sass": "^4.14.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.1.0",
+    "react-router-dom": "^5.2.0",
```

Comment:
Why do we need React Router DOM?

Reply:
I didn't want to add anchor tags, since react pops up errors in the console.So I thought to replace it with Link but then the routes weren't set up so couldn't use it.

Reply:
Ah, anyway we will be needing it, but not now. Let's remove it for now.

Reply:
Sure👍🏻